public/stylesheets/app.css

```diff
@@ -1,146 +1,147 @@
 :root {
-	--bg-color: white;
-	--fg-color: black;
-	--dark-mode-bg-color: #0f0f1b;
-	--dark-mode-fg-color: white;
+  --bg-color: white;
+  --fg-color: black;
+  --dark-mode-bg-color: #0f0f1b;
+  --dark-mode-fg-color: white;
```

Comment:
Looks like your text editor converted hard-tabs in `.css` to 2-space indentation. I try to use hard-tabs for CSS files. Reform from 2-spaces to hard-tabs.